src/StyleSheet.js

```diff
@@ -30,22 +29,26 @@ export default class StyleSheet {
     this.rules = Object.create(null)
     // Used to ensure correct rules order.
     this.rulesIndex = []
-    this.classes = Object.create(null)
+    this.classes = this.options.classes = Object.create(null)
```

Comment:
I'm wondering if there is a better way to implement this. It doesn't feel completely natural having the classes reference on the options object. I feel like the classes should be retrieved from the rules container rather than passing around the object reference like this. Is there ever a scenario where the classes are provided to the RulesContainer? If not, this is a bit of a stretch for the usage of options.